### PR TITLE
LPS-198165 Remove Release FF for import maps and Theme SVG client extensions

### PR DESCRIPTION
**Pull request by chestofwonders**
[LPS-198165](https://liferay.atlassian.net/browse/LPS-198165) The goal is to get rid of two FF: TYPE_JS_IMPORT_MAPS_ENTRY and TYPE_THEME_SPRITEMAP

### CODE REVIEW

**Comment by chestofwonders** [comment]
ci:test:sf

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/css.jsp [hunk 1]
```diff
@@ -35,19 +35,6 @@ LayoutLookAndFeelDisplayContext layoutLookAndFeelDisplayContext = new LayoutLook
 	/>
 </liferay-frontend:fieldset>
 
-<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPS-166479") %>'>
-	<liferay-frontend:fieldset
```
Comment (dsanz) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/css.jsp:39]: This UI bit about the fieldset, and any other UI subject to the FF existence has to be kept. Removing FF means indeed removing the FF checks, therefore, logic/UI which was being gated under such check is now unconditionally applied.

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout_set/css.jsp [hunk 2]
```diff
@@ -35,19 +35,6 @@ LayoutSet selLayoutSet = layoutsAdminDisplayContext.getSelLayoutSet();
 	/>
 </liferay-frontend:fieldset>
 
-<c:if test='<%= FeatureFlagManagerUtil.isEnabled("LPS-166479") %>'>
-	<liferay-frontend:fieldset
```
Comment (dsanz) [inline modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout_set/css.jsp:39]: same here

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase [hunk 3]
```diff
@@ -1194,32 +1194,6 @@ definition {
 			locator1 = "TextInput#ANY");
 	}
 
-	@description = "LPS-166479. Created (theme SVG extensions), the theme SVG extensions admin page lists the name and type"
-	@priority = 4
-	test ThemeSvgTypeCanBeCreated {
-		property custom.properties = "feature.flag.LPS-166479=true";
```
Comment (dsanz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase:1200]: Here we would like to keep the test but remove the declaration of the additional property, so I think (@CarlosBrichete please confirm) that removing only line 1200 is enough. The rest of the test has to remain

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase [hunk 4]
```diff
@@ -36,53 +36,9 @@ definition {
 		}
 	}
 
-	@description = "LPS-166479. Verify that new SVG spritemap is applied on the page"
-	@priority = 3
-	test CanExtendSvgFileFromDM {
-		property custom.properties = "feature.flag.LPS-166479=true";
```
Comment (dsanz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:42]: Same considerations here

**Review by dsanz — CHANGES_REQUESTED** [review]

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase (hunk 3 above)
Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase:1200]: Yes, if we are removing the FF we can just get rid off that property and the test should keep working as before (not only that, the test should keep working removing the property or keeping it, it doesn't matter)

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase (hunk 4 above)
Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:42]: Same here

File: modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/css.jsp (hunk 1 above)
Comment (chestofwonders) [reply modules/apps/layout/layout-admin-web/src/main/resources/META-INF/resources/layout/css.jsp:39]: So, here would only be necessary to remove lines 38 and 49? @dsanz @CarlosBrichete

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase (hunk 3 above)
Comment (chestofwonders) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase:1200]: It's seems that custom.properties is not being used, so looks safe to remove it, doesn't it? but...for what it was been used when FF was enabled? (just curiosity) @dsanz @CarlosBrichete

Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtension.testcase:1200]: custom.properties is for the portal-ext.properties file but in CI. When the FF is removed from the code and you still keep the property in the test, it just does nothing, so it won't break the test

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase [hunk 5]
```diff
@@ -36,10 +36,9 @@ definition {
 		}
 	}
 
-	@description = "LPS-166479. Verify that new SVG spritemap is applied on the page"
+	@description = "Verify that new SVG spritemap is applied on the page"
 	@priority = 3
```
Comment (dsanz) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:40]: @CarlosBrichete usually you keep the LPS number in the test description right? if that's the case, then this one should be kept

Comment (CarlosBrichete) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:40]: Yes, we do. It is important to have the LPS in order to have the tests tracked with their parent story.

Comment (dsanz) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:40]: ok then @chestofwonders pls re-add this LPS number, I think after this change we're ready to go

Comment (chestofwonders) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/remotewebapps/ClientExtensionThemeSvg.testcase:40]: it's done @dsanz @CarlosBrichete, if we are ok, we can go with it

**Review by dsanz — APPROVED** [review]
LGTM ;)

**Comment by dsanz** [comment]
@chestofwonders please review ClientExtension.testcase file. I think there is an extra blank line between `test ThemeSvgTypeCanBeCreated {` and the first `task`

@CarlosBrichete can it be the cause of the SF failing?

**Comment by chestofwonders** [comment]
ci:test:sf

**Comment by chestofwonders** [comment]
ci:forward

**Comment by dsanz** [comment]
@CarlosBrichete could you pls take a look to relevant? I see many "Build was aborted", thx

**Comment by CarlosBrichete** [comment]
@dsanz I'd say the failures are not related, it seems to be something related with a punctual failure in CI. If you want to be more sure we can run it again, but if you are in a hurry to merge this I would say we are safe to forward it

**Comment by dsanz** [comment]
ci:test:relevant

**Comment by chestofwonders** [comment]
@dsanz  @CarlosBrichete, relevant has been failed again (and stable, which was success previously). Do you think I can forward safely?

**Comment by bryceosterhaus** [comment]
@chestofwonders feel free to forward. QA said it seems unrelated and also with the nature of the change, just removing FF, I think we are pretty safe here.

**Comment by chestofwonders** [comment]
ci:forward:force

**Comment by chestofwonders** [comment]
ci:close

**Comment by chestofwonders** [comment]
Continues [here](https://github.com/liferay-frontend/liferay-portal/pull/3757)

**Pull request closed without merging**